onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -68,6 +68,17 @@ namespace OperatorHelper
         originalValues = std::move(expanded);
     }
 
+    uint32_t GetBitMaskFromIndices(gsl::span<const uint32_t> indices) noexcept
```

Comment:
>GetBitMaskFromIndices [](http://example.com/codeflow?start=13&length=21)

I don't want to duplicate this mini-helper function (which is also found in DmlCommon.cpp), but I'm stuck by this directory's weird isolation in that it doesn't include anything else DML-ish outside its scope. 🤔.